File: LLM-Translations/GPT-4/codenet/Python/C/s103354654.c
#include <stdio.h>

int main() {
    int W, H, x, y, r;
    scanf("%d %d %d %d %d", &W, &H, &x, &y, &r);

    if (x >= r && (W - r) >= x) {
        if (y >= r && (H - r) >= y) {
            printf("Yes\n");
        } else {
            printf("No\n");
        }
    } else {
        printf("No\n");
    }

    return 0;
} // 

File: LLM-Translations/GPT-4/codenet/Python/Go/s103354654.go
package main

import (
	"fmt"
)

func main() {
	var W, H, x, y, r int
	fmt.Scan(&W, &H, &x, &y, &r)

	if x >= r && (W-r) >= x {
		if y >= r && (H-r) >= y {
			fmt.Println("Yes")
		} else {
			fmt.Println("No")
		}
	} else {
		fmt.Println("No")
	}
}

// 
